app/graphql/mutations/profile/edit.rb

```diff
@@ -16,7 +19,7 @@ class Edit < BaseMutation
 
       def resolve(args = {})
         profile = authorized_profile(args)
-        profile.update(args.compact)
```

Comment:
There is nothing on the profile itself that is editable by our APIs after this change.